osu.Game/Storyboards/Storyboard.cs

```diff
@@ -104,7 +104,13 @@ public Drawable CreateSpriteFromResourcePath(string path, TextureStore textureSt
                 drawable = new Sprite { Texture = textureStore.Get(storyboardPath) };
             // if the texture isn't available locally in the beatmap, some storyboards choose to source from the underlying skin lookup hierarchy.
             else if (UseSkinSprites)
-                drawable = new SkinnableSprite(path);
+            {
+                drawable = new SkinnableSprite(path)
+                {
+                    RelativeSizeAxes = Axes.None,
+                    AutoSizeAxes = Axes.Both,
```

Comment:
This is potentially a scary amount of autosize and one would hope that this is a simple automatic operation that `SkinableSprite` does when no size/relative size is provided by just taking the `Sprite`'s size...

But I guess this is fine?

Reply:
Yeah I'm not too sure about this myself, went with how it's done in other places like [`CatchComboDisplay`](https://github.com/ppy/osu/blob/95891bc6550180c5a3959ebcb3122a8712e8da2d/osu.Game.Rulesets.Catch/UI/CatcherArea.cs#L58-L61) and [`DrawableNote`](https://github.com/ppy/osu/blob/f9d5abff8a8d3b6752343c185eb7767a3944a532/osu.Game.Rulesets.Mania/Objects/Drawables/DrawableNote.cs#L55-L59). The `RelativeSizeAxes = Axes.Both` specification in `SkinnableDrawable` would definitely have to be removed first to allow such automatic operation in place.

Reply:
Doesn't seem to have a noticeable effect in an edge case scenario I've got.